src/pymortests/demos.py

```diff
@@ -13,7 +13,9 @@
 from pymortests.base import runmodule, check_results
 from pymor.core.exceptions import PySideMissing
 from pymor.gui.qt import stop_gui_processes
+from pymor.core.config import is_windows_platform
 
+pytestmark = pytest.mark.skipif(is_windows_platform(), reason='interpreter crashes')
```

Comment:
The ipython tests just endless stall. Every other demo threw access violations _after_ actual computations. They work fine in my VM though, so it's appveyor setup specific afaict. And we could tighten the skip to that.